var/spack/repos/builtin/packages/catalyst/package.py

```diff
@@ -118,16 +138,44 @@ def do_stage(self, mirror_only=False):
             tty.msg("Already generated %s in %s" % (self.name,
                                                     self.stage.path))
 
+    @when('@:5.4.1')
     def setup_environment(self, spack_env, run_env):
+        # paraview 5.4 and earlier
+        # - libs  under lib/paraview-5.4
         if os.path.isdir(self.prefix.lib64):
             lib_dir = self.prefix.lib64
         else:
             lib_dir = self.prefix.lib
-        paraview_version = 'paraview-%s' % self.spec.version.up_to(2)
-        run_env.prepend_path('LIBRARY_PATH', join_path(lib_dir,
-                             paraview_version))
-        run_env.prepend_path('LD_LIBRARY_PATH', join_path(lib_dir,
-                             paraview_version))
+
+        run_env.set('ParaView_DIR', self.prefix)
+
+        # Everything else under lib/paraview-5.4
+        lib_dir = join_path(lib_dir, paraview_subdir)
+        run_env.prepend_path('LIBRARY_PATH', lib_dir)
+        run_env.prepend_path('LD_LIBRARY_PATH', lib_dir)
+
+    def setup_environment(self, spack_env, run_env):
+        # paraview 5.5 and later
+        # - cmake under lib/cmake/paraview-5.5
+        # - libs  under lib
+        # - python bits under lib/python2.8/site-packages
+        if os.path.isdir(self.prefix.lib64):
+            lib_dir = self.prefix.lib64
+        else:
+            lib_dir = self.prefix.lib
+
+        run_env.set('ParaView_DIR', self.prefix)
+        run_env.prepend_path('LIBRARY_PATH', lib_dir)
+        run_env.prepend_path('LD_LIBRARY_PATH', lib_dir)
```

Comment:
This probably isn't necessary, it is already added by:
https://github.com/spack/spack/blob/develop/etc/spack/defaults/modules.yaml#L29,L34